bin/install-ci-tools

```diff
@@ -2,7 +2,7 @@
 set -xeuo pipefail
 
 KIND_VERSION="0.7.0"
-HELM_VERSION="2.16.1"
+HELM_VERSION="3.1.3"
```

Comment:
I think we also need
```
helm repo add stable https://kubernetes-charts.storage.googleapis.com
```
somewhere in this file

Reply:
fixed.